### PR TITLE
Add Plotter.center and Plotter.image examples

### PR DESCRIPTION
**Pull request by Arnav-2004**
### Overview

Adding examples for module `pyvista.plotting.plotting`
Issue: #1629

### Details

- Added `Plotter.center` example
- Added `Plotter.image` example

### CODE REVIEW

**Comment by Arnav-2004** [comment]
About the failing check, do I add `# doctest:+SKIP`

**Comment by tkoyama010** [comment]
@Arnav-2004 Yes. Please. Thank you for contribution.

File: pyvista/plotting/plotter.py [hunk 1]
```diff
@@ -1778,7 +1793,22 @@ def _make_render_window_current(self):
 
     @property
     def image(self):  # numpydoc ignore=RT01
-        """Return an image array of current render window."""
+        """Return an image array of current render window.
+
+        Returns
+        -------
+        numpy.ndarray
```
Comment (tkoyama010) [inline pyvista/plotting/plotter.py:1800]: @Arnav-2004 Thanks for this. We get `pyvista.pyvista_ndarray` in my mind.
```suggestion
        pyvista.pyvista_ndarray
```
You can check it using type function.
```python
>>> type(pl.image)
pyvista.core.pyvista_ndarray.pyvista_ndarray
```

File: pyvista/plotting/plotter.py [hunk 2]
```diff
@@ -1575,7 +1575,22 @@ def length(self):  # numpydoc ignore=RT01
 
     @property
     def center(self):  # numpydoc ignore=RT01
-        """Return the center of the active renderer."""
+        """Return the center of the active renderer.
+
+        Returns
+        -------
+        list[float, float, float]
```
Comment (tkoyama010) [inline pyvista/plotting/plotter.py:1582]:
```suggestion
        list[numpy.float64, numpy.float64, numpy.float64]
```
```python
>>> type(pl.center)
list
>>> type(pl.center[0])
numpy.float64
>>> type(pl.center[1])
numpy.float64
>>> type(pl.center[2])
numpy.float64
```

**Review by tkoyama010 — CHANGES_REQUESTED** [review]

File: pyvista/plotting/plotter.py (hunk 2 above)
Comment (Arnav-2004) [reply pyvista/plotting/plotter.py:1582]: In that case wouldn't `Plotting.bounds` also return `tuple[numpy.float64, numpy.float64, numpy.float64, numpy.float64, numpy.float64, numpy.float64]`

```py
>>> type(pl.bounds[0]) 
<class 'numpy.float64'>
```

Comment (tkoyama010) [reply pyvista/plotting/plotter.py:1582]: @Arnav-2004 Good catch. Could you also fix it too?

File: pyvista/plotting/plotter.py (hunk 1 above)
Comment (Arnav-2004) [reply pyvista/plotting/plotter.py:1800]: `Plotter.get_image_depth` also returns `pyvista.core.pyvista_ndarray.pyvista_ndarray` but has been documented as `numpy.ndarray`

```py
>>> type(zval)
<class 'pyvista.core.pyvista_ndarray.pyvista_ndarray'>
```

Comment (Arnav-2004) [reply pyvista/plotting/plotter.py:1800]: Same with `Plotter.screenshot`

**Review by tkoyama010 — APPROVED** [review]
LGTM

**Pull request merged**